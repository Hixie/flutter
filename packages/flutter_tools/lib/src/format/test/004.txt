class foo {
  foo(this.bar());
  dynamic bar;
}

class baz {
  void foo(x) { }
  void bar() { }

  void test() {
    foo(this.bar());
  }
}

void main() { }
