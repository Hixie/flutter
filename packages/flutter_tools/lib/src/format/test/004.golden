class foo {
  foo(dynamic this.bar());

  dynamic bar;
}

class baz {
  void foo(dynamic x) { }

  void bar() { }

  void test() {
    foo(this.bar());
  }
}

void main() { }
